src/tracer/tracer_example.F90

```diff
@@ -181,11 +182,12 @@ subroutine USER_initialize_tracer(restart, day, G, GV, h, diag, OBC, CS, &
   if (.not.restart) then
     if (len_trim(CS%tracer_IC_file) >= 1) then
 !  Read the tracer concentrations from a netcdf file.
-      if (.not.file_exists(CS%tracer_IC_file, G%Domain)) &
-        call MOM_error(FATAL, "USER_initialize_tracer: Unable to open "// &
+      if (.not.file_exists(CS%tracer_IC_file)) &
+        call MOM_error(FATAL, "USER_initialize_tracer: Unable to find "// &
                         CS%tracer_IC_file)
       do m=1,NTR
         call query_vardesc(CS%tr_desc(m), name, caller="USER_initialize_tracer")
+        if (m == 1) &
```

Comment:
Is this a bug?  It looks like only the first tracer is being read.

Reply:
Looks like it. I've removed it.